logs/handler.go

```diff
@@ -0,0 +1,431 @@
+package logs
+
+import (
+	"context"
+	"encoding/json"
+	"log"
+	"net"
+	"net/http"
+	"net/http/httputil"
+	"net/url"
+	"regexp"
+	"strconv"
+	"time"
+
+	"github.com/gorilla/websocket"
+
+	"github.com/openfaas/faas-provider/httputils"
+)
+
+var upgrader = websocket.Upgrader{} // use default options
```

Comment:
What if this was initialised closer to where it was used instead of being in a broader scope?

Reply:
I have removed the websocket implementation